style.css

```diff
@@ -113,3 +136,222 @@ body {
   height: 20px;
   width: 14.7px;
 }
+
+.mobile-version {
+  display: grid;
+}
+
+.second-phase {
+  margin: 114px 24px 0;
+}
+
+.second-page {
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px 16px 28px;
+  display: flexbox;
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest using CSS Flexbox properties on the second-page class which represents each card of the work section. The css property value of flexbox is wrong and nothing will be applied on the html as seen in the image below:


The correct way to apply the property would be:

```
.second-page {
    display: flex;
    flex-driection: column: /* To align the main axis vertically */ 
}
```